.github/workflows/helm-chart-ci.yaml

```diff
@@ -114,8 +106,6 @@ jobs:
 
     needs: [lint-chart]
 
-    if: needs.lint-chart.outputs.changed == 'true'
```

Comment:
we could always reverse it, to always run by default and have option not to run when not needed, but I can't think of the case that it would support that scenario

Reply:
Yeah. was looking for a case but hadn't found one, so decided to just remove it all for now.

Reply:
I think it is acceptable to always check the lint-chart output, even if the output doesn't change.  The primary concerns I have for skipping them are

- In a "skipping" configuration, there are always two reasons the tests were skipped; the tests would have reported the same results AND the tests crashed.  Inability to differentiate the two can hide issues in the testing framework until long after the testing framework was modified.
- Should we want to aggregate pass / fail statistics in testing, the counts will vary widely due to "skipping" tests.  This makes future analysis difficult, as to "smooth" the time series graph, one needs to resort to one of the following undesirable choices
  - Add in the "skipped" test count in scenarios where the tests were missing, to bring the number up to it's "if it was tested" count. (possibly creating a false report, if the testing framework was updated, breaking the tests)
  - Don't add in the skipped test counts, creating spikes in testing that could be misinterpreted as temporary improvements, or as noise, that needs to be filtered in processing quality metrics trends.

If this testing is very expensive, then please provide feedback so my suggestions above don't foul the production pipeline.

Reply:
My only concern was that over time the portfolio of test would be growing and that means the duration for the green checkmark would be getting longer, even if there is a trivial change (e.g. spelling correction). But I support your point of view as well.